### ISSUE TITLE
Does it work on Ubuntu?

### ISSUE THREAD

**Issue by j0ker70** [stoeckmann/xwallpaper#39]
I am running Ubuntu 20.04 and trying to change my wallpaper using xwallpaper. But it is not able to change my wallpaper though the command runs without an error. 
`xwallpaper --zoom dracula-red.png`

Is there anything I can do?

**Comment by stoeckmann** [comment]
Hi @j0ker70,

xwallpaper is supported by Ubuntu. In fact, Ubuntu ships xwallpaper packages.

The problem is most likely your window manager. Do you use Gnome? Or KDE? Perhaps Enlightenment? These desktop environments handle their wallpapers "on their own", i.e. they do not access the pixmap of X's root window.

Or do you use X at all? If you are running a wayland compositor like sway etc. then xwallpaper also has no real effect. It might set the wallpaper of Xwayland, but that's not what a user would see.

So, in short: Can you tell me which window manager you are using?

**Comment by j0ker70** [comment]
I am using Gnome

**Comment by stoeckmann** [comment]
In that case xwallpaper does not work, unfortunately. :(
Or let's say: It does what it is supposed to do but it does not do what you would expect.

**Comment by j0ker70** [comment]
Ow, Its okay then. I will try to find some alternatives.
